sources/packages/web/src/components/aest/institution/modals/AssessOfferingChangeModal.vue

```diff
@@ -0,0 +1,100 @@
+<template>
```

Comment:
In the future, we can have a generic Approve/Decline modal with a note.
@ann-aot @guru-aot @henryoforeh-aot